docs/basics/getting-started.md

```diff
@@ -1,210 +1,159 @@
 
 ```shell
-cross-seed search
+cross-seed gen-config
 ```
 
-:::tip
-Any CLI argument provided will override the settings used in your `config.js`
-:::
+This will create a basic `config.js` file at `~/.cross-seed/config.js` or
+`AppData\Local\cross-seed\config.js`. You can then edit this file to add the
+necessary configurations.
+
+#### For Docker users:
 
-For CLI commands, a basic example is below:
+Docker users can generate the config file by running the following command:
 
 ```shell
-# one liner
-cross-seed search --torrent-dir /path/to/dir/with/.torrent/files --output-dir . --torznab https://localhost/prowlarr/1/api?apikey=12345
-# readable
-cross-seed search \
-  --torrent-dir /path/to/dir/with/torrent/files \ # folder containing .torrent files
-  -o . \ # any directory, cross-seed will put its output torrents here
-  --torznab https://localhost/prowlarr/1/api?apikey=12345 # any valid torznab link, separated by spaces
+docker run \
+-v /path/to/config:/config \
+ghcr.io/cross-seed/cross-seed gen-config
 ```
 
-If you were lucky, you've already got your first cross-seed in your current
-directory!
-
-:::info
-`cross-seed`, by default, is particular about the contents of torrents you feed
-it. It will ignore torrents with non-video files in them, and it will ignore
-torrents that look like single episodes.
-
-Use the command line flags to override any defaults:
-`--include-episodes --include-non-videos`
-
-These command line arguments are always configurable in the config.js file (generated with gen-config).
-You won't have to specify them if they are set unless you wish to override the settings temporarily.
+This will create a `config.js` file inside the `/config` directory you
+specified. You’ll need to open this file and edit it as described below.
 
-Below we will cover the configuration process.
-:::
+---
 
-## Scaling Up
+### 3. Edit the Config File
+
+Once you’ve created your config file, you’ll need to fill in the necessary
+fields to connect to your torrent client and indexer. At a minimum, you should
+configure the following:
+
+-   **`torznab`**: URLs of your Torznab indexers (from Prowlarr or Jackett).
+-   **`torrentDir`**: The directory where your .torrent files are located (from
+    your torrent client).
+-   **`outputDir`**: Create a new directory somewhere to house the torrent files `cross-seed` outputs.
+-   **Connection to your torrent client**: Depending on your client, you'll need
+    to specify one of the following:
+    -   `rtorrentRpcUrl`: For rTorrent users. Often looks like `http://user:pass@localhost:8080/RPC2`
+    -   `qbittorrentUrl`: For qBittorrent users. Often looks like `http://user:pass@localhost:8080`
+    -   `transmissionRpcUrl`: For Transmission users. Often looks like `http://user:pass@localhost:9091/transmission/rpc`
+    -   `delugeRpcUrl`: For Deluge users. Often looks like `http://:pass@localhost:8112/json` (the colon before pass is intentional)
```

Comment:
I've tried to cross-reference via hyperlinks any place where options and general concepts are mentioned, specifically options, so I would propose the same is done here.

Just linking to the options, page, which would give more details without the user having to "dig" for them.